xvelmap/xr_accessor.py

```diff
@@ -53,6 +55,7 @@ def plot(self, u_var, v_var, lat_dim='latitude', lon_dim='longitude',
             Display options for the interactive map.
 
         """
+        _ds = xr.Dataset()
```

Comment:
Alternatively you could do `ds = self._ds.copy()`. I think it's easier. Copying a dataset (without the underlying data) is cheap, and you could just use `ds` everywhere below leaving the input dataset untouched and without the need to rename latitude/longitude.